cves/kernel/CVE-2021-38166.yml

```diff
@@ -128,10 +128,10 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: This code did not have unit tests when it was originally committed to the kernel.
+  fix: false
+  fix_answer: The fix didn't add any unit tests, it simply fixed the issue and moved on.
```

Comment:
You might want to include some details on how you came to this conclusion. Was there no mention in the mailing list of additional unit tests added? What about commits to related unit tests in the repository itself?